### PR TITLE
Update NotificationPushsafer.cpp

### PR DESCRIPTION
**Pull request by appzer**
Fix image url issue (occured an error when there is no image url)

### CODE REVIEW

File: notifications/NotificationPushsafer.cpp [hunk 1]
```diff
@@ -77,22 +77,27 @@ bool CNotificationPushsafer::SendMessageImplementation(
 			PS_t = cSubject;
 		}
 
-		if (HTTPClient::GETBinary(CURLEncode::URLDecode(PS_p), ExtraHeadersBinary, camimage, 10))
+		if (PS_p != "")
```
Comment (gizmocuz) [inline notifications/NotificationPushsafer.cpp]: can you change this to

`is (!PS_p.empty())`

File: notifications/NotificationPushsafer.cpp [hunk 2]
```diff
@@ -77,22 +77,27 @@ bool CNotificationPushsafer::SendMessageImplementation(
...
-			sPostData << "&k=" << _apikey << "&d=" << PS_d << "&i=" << PS_i << "&s=" << PS_s << "&v=" << PS_v << "&t=" << PS_t << "&m=" << PS_m << "&p=data:image/jpeg;base64," << base64ImageString << "&u=" << PS_u << "&ut=" << PS_ut << "&l=" << PS_l;
+			if (HTTPClient::GETBinary(CURLEncode::URLDecode(PS_p), ExtraHeadersBinary, camimage, 10))
+			{
+				std::string base64ImageString(camimage.begin(), camimage.end());
+				base64ImageString = base64_encode((const unsigned char*)base64ImageString.c_str(), base64ImageString.size());
+				sPostData << "&k=" << _apikey << "&d=" << PS_d << "&i=" << PS_i << "&s=" << PS_s << "&v=" << PS_v << "&t=" << PS_t << "&m=" << PS_m << "&p=data:image/jpeg;base64," << base64ImageString << "&u=" << PS_u << "&ut=" << PS_ut << "&l=" << PS_l;
+			}
+			else
+			{
+				_log.Log(LOG_STATUS, "Pushsafer: can't download image > wrong url");
```
Comment (gizmocuz) [inline notifications/NotificationPushsafer.cpp]: I think this should be LOG_ERROR

File: notifications/NotificationPushsafer.cpp [hunk 3]
```diff
@@ -101,7 +106,7 @@ bool CNotificationPushsafer::SendMessageImplementation(
 		}
 		else
 		{
-			_log.Log(LOG_ERROR, "Pushsafer: can't download image > wrong url");
+			_log.Log(LOG_STATUS, "Pushsafer: can't download image > wrong url");
```
Comment (gizmocuz) [inline notifications/NotificationPushsafer.cpp]: I think this should be LOG_ERROR

File: notifications/NotificationPushsafer.cpp (hunk 1 above)
Comment (appzer) [reply notifications/NotificationPushsafer.cpp]: changed this

File: notifications/NotificationPushsafer.cpp (hunk 2 above)
Comment (appzer) [reply notifications/NotificationPushsafer.cpp]: changed this

File: notifications/NotificationPushsafer.cpp (hunk 3 above)
Comment (appzer) [reply notifications/NotificationPushsafer.cpp]: changed this

**Comment by gizmocuz** [comment]
Thanks!

**Pull request merged**